apps/wallet-mobile/src/yoroi-wallets/cardano/byron/ByronWallet.ts

```diff
@@ -1149,15 +1149,20 @@ export class ByronWallet implements YoroiWallet {
     return this._collateralId
   }
 
-  getCollateralInfo(): {utxo: RawUtxo | undefined; amount: Balance.Amount; collateralId: string} {
+  getCollateralInfo() {
     const utxos = utxosMaker(this._utxos)
-    const collateralUtxo = utxos.findById(this.collateralId)
+    const collateralId = this.collateralId
+    const collateralUtxo = utxos.findById(collateralId)
     const quantity = collateralUtxo?.amount !== undefined ? asQuantity(collateralUtxo?.amount) : Quantities.zero
```

Comment:
I think you can go with directly 
```suggestion
    const quantity = asQuantity(collateralUtxo?.amount)
```

Reply:
Typescript is complaining, but the `?.` in `collateralUtxo?.amount` should not be there 👍 
<img width="1051" alt="Screenshot 2024-01-05 at 15 21 18" src="https://github.com/Emurgo/yoroi/assets/19684576/b9203ebd-018b-468d-a653-e606af3fa48a">

Reply:
aaaa, I refactored on the Portfolio but didn't merge it yet, so nevermind